content/raw/chat/activity_center.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "activity center spec"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Activity Center
+
+## Designs
+
+- [designs](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5358%3A5666)
+
+## Use Cases
+
+TODO
+
+## Functional Requirements
+
+### General
+
+* The Activity Center MUST support filtering by 4 categories: "All", "Mentions", "Replies", "Contact Requests"
+* The Activity Center MAY contain the following type of items:
+    * contact request
+    * mention in a community
+    * mention in public chat
+    * mention in a group
+    * mentions in a 1:1
+    * reply to a message from the user
+* Clicking the "Mark all as read" MUST mark all activity items read of the selected category
```

Comment:
If all items are marked as read, what happens to contact requests items?

Reply:
@iurimatias good point, well spotted!  If all items are marked as read, this action should only mark messages as read, this action should not impact any pending contact requests or identity verification requests.